src/vcpkg/vcpkgpaths.cpp

```diff
@@ -1261,24 +1258,17 @@ namespace vcpkg
                     expected_right_tag};
         }
 
-        auto untar = Command{get_tool_exe(Tools::CMAKE)}.string_arg("-E").string_arg("tar").string_arg("xf").path_arg(
-            git_tree_temp_tar);
-
-        auto untar_output = cmd_execute_and_capture_output(untar, WorkingDirectory{git_tree_temp});
+        extract_tar_cmake(get_required_tool_exe(Tools::CMAKE), git_tree_temp_tar, git_tree_temp);
         // Attempt to remove temporary files, though non-critical.
         fs.remove(git_tree_temp_tar, IgnoreErrors{});
-        if (untar_output.exit_code != 0)
-        {
-            return {Strings::format("cmake's untar failed with message:\n%s", untar_output.output), expected_right_tag};
-        }
 
         std::error_code ec;
-        fs.rename(git_tree_temp, git_tree_final, ec);
-
+        fs.rename_with_retry(git_tree_temp, git_tree_final, ec);
```

Comment:
It really seems like we should just get rid of `fs.rename` and only have `_with_retry`.

No changes requested to this PR.